presto-main-base/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -102,12 +102,25 @@ public class TaskManagerConfig
     private Duration highMemoryTaskKillerFrequentFullGCDurationThreshold = new Duration(1, SECONDS);
     private double highMemoryTaskKillerHeapMemoryThreshold = 0.9;
     private boolean enableEventLoop;
-    private Duration slowMethodThresholdOnEventLoop = new Duration(10, SECONDS);
+    private Duration slowMethodThresholdOnEventLoop = new Duration(5, MINUTES);
+    private boolean slowMethodLoggingEnabled;
 
-    @Min(50_000_000L)
+    public boolean getSlowMethodLoggingEnabled()
```

Comment:
do we need new config ? can we assume slowMethodThresholdOnEventLoop > 0 === ON and slowMethodThresholdOnEventLoop == 0 is OFF ?